kasa/cli.py

```diff
@@ -345,6 +345,9 @@ async def brightness(dev: SmartBulb, brightness: int, transition: int):
 async def temperature(dev: SmartBulb, temperature: int, transition: int):
     """Get or set color temperature."""
     await dev.update()
+    if not dev.is_variable_color_temp:
+        click.echo(f"Device does not support color temperature")
```

Comment:
```suggestion
        click.echo("Device does not support color temperature")
```

Reply:
Is this not also necessary in line 320?
`    if not dev.is_dimmable:
        click.echo("This device does not support brightness.")`

Reply:
Hmm, sorry, I don't understand what you mean as that line has no f-string (`f"something"`)?